src/hiero_sdk_python/account/account_create_transaction.py

```
@@ -40,7 +45,7 @@ def __init__(
         alias: Optional[EvmAddress] = None,
         staked_account_id: Optional[AccountId] = None,
         staked_node_id: Optional[int] = None,
-        decline_staking_reward: Optional[bool] = False
+        decline_staking_reward: Optional[bool] = False,
```

Ruff (0.14.14)

[error] Boolean-typed positional argument in function definition

(FBT001)